### PR TITLE
Add Corona Data Scraper project data and loader script

### PR DESCRIPTION
**Pull request by willf**
To quote the [Corona Data Scraper](https://coronadatascraper.com) web page,

> Corona Data Scraper pulls COVID-19 Coronavirus case data from verified sources, finds the corresponding GeoJSON features, and adds population data.

We transform this data for use in the Covasim parameter format. It is stored
in CSV-format. 

The following columns are present in the data:

- name: Apparently, a unique name for the data set
- level: level of the data (country, state, county, city)
- city: The city name
- county: The county or parish
- state: The state, province, or region
- country: [ ISO 3166-1 alpha-3 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3)
- population: The estimated population of the location
- lat: latitude
- long: longitude
- url: Data source
- aggregate
- tz: Time zone
- recovered: Cumulative number recovered (as of date)
- active: Cumulative number active (as of date)
- growthFactor:
- date: Date in yyyy-MM-dd text format
- day: Number of days since first reporting
- positives: Cumulative number of positive cases (as of date)
- death: Cumulative number of deaths (as of date)
- tests: Cumulative number of tests (as of date)
- new_positives: New positives on this date
- new_active: New active cases on this date
- new_death: Number of deaths on this date
- new_tests: New tests on this date

As of April 4, 2020, There are apparently 3280 data sets.

Data is in the `data` directory.  
Loader is in the `data_loader` directory.

Partial completion of https://github.com/InstituteforDiseaseModeling/covasim/issues/43

### CODE REVIEW

**Comment by willf** [comment]
This is probably a work in progress, but the data is useful right now.

**Comment by willf** [comment]
Replacing with simpler PR without data

File: .gitignore [hunk 1]
```diff
@@ -7,7 +7,7 @@ covasim/cova_seattle/capacity_analysis
 covasim/cova_seattle/school_analysis
 covasim/cova_seattle/seattle_analysis
 results/
-
+data/
```
Comment (gwincr11) [inline .gitignore:10]: I was wondering if we wanted to seperate the population data from the virus data?
https://github.com/InstituteforDiseaseModeling/covasim/blob/e1707f931011ca3c6bf72598f3c371ec53036259/.gitignore#L142-L144

File: data_loaders/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+We transform this data for use in the Covasim parameter format. It is stored
+in CSV-format. 
+
+### Updating
+
+To update the  Corona Data Scraper data,
+
+```bash
+python data_loaders/load_corona_data_scraper_data.py
```
Comment (gwincr11) [inline data_loaders/README.md:18]: I was considering what a unified single file that could look like here: https://github.com/InstituteforDiseaseModeling/covasim/blob/e1707f931011ca3c6bf72598f3c371ec53036259/covasim/datasets/update_sources.py

File: data_loaders/load_corona_data_scraper_data.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import pandas as pd
+import os.path
+import sys
+import logging
+
+log = logging.getLogger("Corona Data Scraper Data Loader")
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
```
Comment (gwincr11) [inline data_loaders/load_corona_data_scraper_data.py:7]: Yeah Logging!! ❤️

**Pull request closed without merging**